crates/control/src/visual_referee_filter.rs

```diff
@@ -97,12 +99,13 @@ impl VisualRefereeFilter {
 
             // Initially a random visual referee decision
             let gesture =
-                VisualRefereeDecision::from_u32(self.random_state.gen_range(1..=13)).unwrap();
+                GestureVisualRefereeDecision::from_u32(self.random_state.gen_range(1..=13))
+                    .unwrap();
```

Comment:
Do we still need to send these gesture referee messages?

Reply:
Are you sure about this? We expect the rules to include more visual referee stuff in the future right? We can remove it in a separate PR however, so that we could easily revert it in theory.